docs/section5/Rule5-37.md

```diff
@@ -0,0 +1,81 @@
+
+# Envelope - Rule 5-37  
+
+**Rule ID:** 5-37  
+**Rule Description:** Skylight U-factors for residential, non-residential and semi-heated spaces in the baseline model must match the appropriate requirements in Table G3.4-1 through G3.4-8.  
+**Rule Assertion:** B-RMR subsurface: U_factor = expected value  
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-5(e) Building Envelope Modeling Requirements for the Baseline building
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+**Function Call:**
+
+  1. get_surface_conditioning_category()
+  2. get_opaque_surface_type()
+  3. get_building_segment_skylight_roof_ratios()
+  4. data_lookup()
+
+## Rule Logic:  
+
+- Get building climate zone: `climate_zone = B_RMR.weather.climate_zone`  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dictionary_b = get_surface_conditioning_category(B_RMR)`  
+
+- Get building skylight roof ratios dictionary: `skylight_roof_ratios_dictionary_b = get_building_segment_skylight_roof_ratios(B_RMR)`
+
+- For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`
+
```

Comment:
based on the interpretation, I believe we need to first categorize zones into SCCs and then check SRR for each SCC. @mkarpman - please weigh in.

Reply:
@mkarpman yes, but the function get_building_segment_skylight_roof_ratios(B_RMR) suggests that we're getting the SRR by each building segmeent. though there should be a step to use get_surface_conditioning_cateogory to categorize all roof surfaces into R, NR, SH and get the total roof area for R roofs, NR Roof, SH Roof and then calculate the SRR for each of those. let me know if that is not the intent

Reply:
@supriyagoel Is this covered in Line 41, 47 and 53?
> @mkarpman yes, but the function get_building_segment_skylight_roof_ratios(B_RMR) suggests that we're getting the SRR by each building segmeent. though there should be a step to use get_surface_conditioning_cateogory to categorize all roof surfaces into R, NR, SH and get the total roof area for R roofs, NR Roof, SH Roof and then calculate the SRR for each of those. let me know if that is not the intent

Reply:
@xingcx - yes, I also commented on the function to get SRR by SCC. Once that function is corrected, this part would work.